### PR TITLE
Issue #16 Init Fridge

### LINKED ISSUE #16: Init fridge
when creating a new fridge, no gui elements for adding new elements
image

### PR DESCRIPTION
**Pull request by nathan-n-poon**
when user creates fridge, they can add their first item
close #16

### CODE REVIEW

File: src/pages/fridges/index.tsx [hunk 1]
```diff
@@ -136,7 +137,24 @@ export const Fridges: React.FC<CreateFridgeProps> = ({}) => {
...
+			return (
+				<Box display={'flex'} flexDirection="column" alignItems="center" >
+
+					<Text>This fridge is empty👀</Text>
+					<Button
+						mt={8/4}
+						variant="outline"
+						colorScheme="teal"
+						border="2px"
+						icon={<AddIcon />}
```
Comment (hungnguyen3) [inline src/pages/fridges/index.tsx:149]: remove this!

File: src/pages/fridges/index.tsx [hunk 2]
```diff
@@ -124,6 +124,7 @@ export const Fridges: React.FC<CreateFridgeProps> = ({}) => {
 						Delete Fridge
 					</Button>
 				</Box>
+
```
Comment (hungnguyen3) [inline src/pages/fridges/index.tsx:127]: remove this line

File: src/pages/fridges/index.tsx [hunk 3]
```diff
@@ -136,7 +137,24 @@ export const Fridges: React.FC<CreateFridgeProps> = ({}) => {
...
+						mt={8/4}
+						variant="outline"
+						colorScheme="teal"
+						border="2px"
+						icon={<AddIcon />}
+						onClick={() => {
+							router.push(`/fridges/createFridgeItem?fridgeId=${fridgeId}`)
+						}}
+					>
+					Add First Fridge Item
```
Comment (hungnguyen3) [inline src/pages/fridges/index.tsx:154]: indent this!

**Review by hungnguyen3 — CHANGES_REQUESTED** [review]

File: src/pages/fridges/index.tsx (hunk 1 above)
Comment (nathan-n-poon) [reply src/pages/fridges/index.tsx:149]: removed icon={<AddIcon />}

File: src/pages/fridges/index.tsx (hunk 2 above)
Comment (nathan-n-poon) [reply src/pages/fridges/index.tsx:127]: done

File: src/pages/fridges/index.tsx (hunk 3 above)
Comment (nathan-n-poon) [reply src/pages/fridges/index.tsx:154]: indented `Add First Fridge Item`

**Review by hungnguyen3 — APPROVED** [review]
Looks good! Good job bruh

**Pull request merged**